kitsune/sumo/migrations/0002_initial_data.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+def create_ratelimit_bypass_perm(apps, schema_editor):
+    # First we get or create the content type.
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+    global_permission_ct, created = ContentType.objects.get_or_create(
+        name='global_permission', app_label='sumo')
+
+    # Then we create a permission attached to that content type.
+    Permission = apps.get_model('auth', 'Permission')
+    perm = Permission.objects.create(
+        name='Bypass Ratelimits',
+        content_type=global_permission_ct,
+        codename='bypass_ratelimit')
+
+
+def remove_ratelimit_bypass_perm(apps, schema_editor):
+    Permission = apps.get_model('auth', 'Permission')
+    perm = Permission.objects.filter(codename='bypass_ratelimit').delete()
+
+
+def create_treejack_switch(apps, schema_editor):
+    Switch = apps.get_model('waffle', 'Switch')
+    Switch.objects.create(
```

Comment:
Will this try to create the Switch, and then fail because it already exists if we run this on prod? (Same for the other initial data migrations)